pkg/shell/nav.jsx

```diff
@@ -48,6 +53,9 @@ export class CockpitNav extends React.Component {
         // Click on active menu item (when using arrows to navigate through menu)
         function clickActiveItem() {
             const cur = document.activeElement;
+            console.log("A added line, it might be executed");
+            if (cur === true)
+                uncalled_function();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17475-20220628-064306-0d71ea51-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)